third_party/merge_zip_files.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
nit: move this to `/src` instead. It's not third-party code, but written by us.

Reply:
Done.